linkerd/opencensus/src/lib.rs

```diff
@@ -28,6 +28,7 @@ where
     T::ResponseBody: 'static,
     S: Stream<Item = Span> + Unpin,
 {
+    debug!("span exporter running");
```

Comment:
nit: prefer to capitalize log messages

```suggestion
    debug!("Span exporter running");
```